app/src/main/java/com/globant/example/mentorapp/home/presentation/view/fragment/ListUsersFragment.java

```diff
@@ -0,0 +1,147 @@
+package com.globant.example.mentorapp.home.presentation.view.fragment;
+
+import android.arch.lifecycle.LifecycleFragment;
+import android.arch.lifecycle.Observer;
+import android.arch.lifecycle.ViewModelProviders;
+import android.content.Context;
+import android.os.Bundle;
+import android.support.annotation.Nullable;
+import android.support.v7.widget.DividerItemDecoration;
+import android.support.v7.widget.LinearLayoutManager;
+import android.support.v7.widget.RecyclerView;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.Adapter;
+
+import com.globant.example.mentorapp.MentorApplication;
+import com.globant.example.mentorapp.R;
+import com.globant.example.mentorapp.home.presentation.model.ModelUserEntity;
+import com.globant.example.mentorapp.home.presentation.model.SharedUserViewModel;
+import com.globant.example.mentorapp.home.presentation.model.UsersModel;
+import com.globant.example.mentorapp.home.presentation.presenter.ListUsersPresenter;
+import com.globant.example.mentorapp.home.presentation.view.activity.HomeScreenActivity;
+import com.globant.example.mentorapp.home.presentation.view.adapter.ListUsersAdapter;
+
+import java.util.List;
+
+import javax.inject.Inject;
+
+/**
+ * A placeholder fragment containing a simple view.
+ */
+public class ListUsersFragment extends LifecycleFragment implements ListUsersInterface {
+
+    public SharedUserViewModel model;
+    @Inject
+    public ListUsersPresenter presenter;
+    @Inject
+    protected LinearLayoutManager linearLayoutManager;
+    protected ListUsersAdapter listUsersAdapter;
+    @Inject
+    DividerItemDecoration itemDecoration;
+    private RecyclerView listUsersRecyclerView;
+
+    @Override
+    public void onCreate(@Nullable Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        ((MentorApplication) getActivity().getApplicationContext()).getApplicationComponent().inject(this);
+    }
+
+    @Override
+    public View onCreateView(LayoutInflater inflater, ViewGroup container,
+                             Bundle savedInstanceState) {
+        View view = inflater.inflate(R.layout.fragment_home_screen, container, false);
+        model = ViewModelProviders.of(this).get(SharedUserViewModel.class);
+        if (model.getUsers() == null || model.getUsers().getValue() == null) {
+            presenter.getUsersList(this);
```

Comment:
You're calling the presenter before calling register bus and also, you're not attaching the view to it